core/src/main/scala/kafka/zk/KafkaZkClient.scala

```diff
@@ -51,8 +51,8 @@ import scala.collection.{Map, Seq, mutable}
  * easier to migrate away from `ZkUtils` (since removed). We should revisit this. We should also consider whether a
  * monolithic [[kafka.zk.ZkData]] is the way to go.
  */
-class KafkaZkClient private[zk] (zooKeeperClient: ZooKeeperClient, isSecure: Boolean, time: Time) extends AutoCloseable with
-  Logging with KafkaMetricsGroup {
+class KafkaZkClient private[zk] (zooKeeperClient: ZooKeeperClient, isSecure: Boolean, time: Time, paginateTopics: Boolean)
```

Comment:
minor optional nit:

Can we follow the line split pattern as the other classes, placing each parameter on 1 line?

`ZkAdminManager` does it this way
```scala
class ZkAdminManager(val config: KafkaConfig,
                     val metrics: Metrics,
                     val metadataCache: MetadataCache,
                     val zkClient: KafkaZkClient,
                     val controller: KafkaController) extends Logging with KafkaMetricsGroup {
```